src/cmd/cli/command/deploymentinfo.go

```diff
@@ -20,7 +20,7 @@ func printPlaygroundPortalServiceURLs(serviceInfos []*defangv1.ServiceInfo) {
 	}
 }
 
-func printEndpoints(serviceInfos []*defangv1.ServiceInfo) {
+func printServiceStatesAndEndpoints(serviceInfos []*defangv1.ServiceInfo) {
```

Comment:
```suggestion
// TODO: consolidate with the getServices "defang ls" code.
func printServiceStatesAndEndpoints(serviceInfos []*defangv1.ServiceInfo) {
```

Reply:
Working on this in https://github.com/DefangLabs/defang/pull/1033